SEC/Equities/EquityInstruments.rdf

```diff
@@ -768,29 +859,37 @@
 		<skos:definition xml:lang="en">grants or bestows ownership via contract</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasCoupon">
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasAdjustableDividendRate">
+		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
+		<rdfs:label>has adjustable dividend rate</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;PreferredDividend"/>
+		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;PercentageMonetaryAmount"/>
```

Comment:
This is not adequate to represent a formula.

Reply:
This is just to represent a straight rate.  If one would need to represent a formula, you could use hasExpression on the adjustable rate dividend class, but this property wouldn't be the way to do that.  You would do that on the class,  which one can do as needed.  Do you have counter examples that would show why this additional property is needed?

Reply:
But this is *adjustable* rate so needs reference to some sort of formula by which it's adjusted. the explanatoryNote even says "is set by a predetermined formula "

Reply:
Raised #1138